src/views/LeadAdd/LeadDetail/index.js

```diff
@@ -75,6 +77,26 @@ const AuthorEmptyComponent = () => (
 const FaramBasicSelectInput = FaramInputElement(BasicSelectInput);
 const ModalButton = Modalize(Button);
 
+const capitalize = string => (
+    string.charAt(0).toUpperCase() + string.slice(1).toLowerCase()
+);
```

Comment:
```suggestion
function capitalize(str) {
    return str.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
};
```

No need to check length of string or if it is defined?